src/hww.c

```diff
@@ -97,6 +99,10 @@ static size_t _api_info(uint8_t* buf)
     *current = keystore_is_locked() ? 0x00 : 0x01;
     current++;
 
+    // 1 byte initialized status
+    *current = memory_is_initialized() ? 0x00 : 0x01;
```

Comment:
wouldn't `0x01` for "is_initialized" be more intuitive? If one interprets 0x00 as false and 0x01 as true.

Reply:
True, I just copy and pasted the above without thinking much. I'll change it, and also change the `switch` in the go api then.